spec/spec_helper.rb

```diff
@@ -23,15 +18,12 @@
   config.default_formatter = "doc" if config.files_to_run.one?
   config.order = :random
 
-  config.infer_spec_type_from_file_location!
-  config.filter_rails_from_backtrace!
   Kernel.srand config.seed
 
-  config.include ManageIQ::GraphQL::Engine.routes.url_helpers, :type => :routing
+  config.expose_dsl_globally = true
```

Comment:
* This is a default, you shouldn't need to specify this to true.
* You're explicitly calling context methods within the RSpec namespace (`RSpec.describe`) so don't you actually want this to be *false*?

Reply:
Ah yes! Good catch. I think I added this when I included the shared files from the main repo, but later un-included them when I realized I only needed the factories for integration tests. Will delete.